scripts/deploy_kakarot.py

```diff
@@ -44,7 +44,16 @@ async def main():
     )
     deployments["blockhash_registry"] = await deploy(
         "blockhash_registry",
-        deployments["kakarot"]["address"],
+        deployments["kakarot"]["address"],  # kakarot address
+    )
+    dump_deployments(deployments)
+
+    deployments["EVM"] = await deploy(
+        "EVM",
+        ETH_TOKEN_ADDRESS,  # native_token_address_
+        class_hash["contract_account"],  # contract_account_class_hash_
+        class_hash["proxy"],  # account_proxy_class_hash
+        deployments["blockhash_registry"]["address"],  # blockhash_registry address
     )
     dump_deployments(deployments)
```

Comment:
`dump_deployments` can just be called once after all deployments are done

Reply:
Makes sense, I followed the way it was being done in the project, but lemme refactor that!